File: Cup/Compiler-Do-While/Examples/sum.c
int sum(int n) {
    int s;
    s = 0;
    do {
        s = s + n;
        n = n - 1;
    } while (n != 0);
    return s;
}

int main() {
    int n;
    n = getchar();
    n = n - 48;
    putchar(sum(n) + 48);
    putchar(10);
}

File: Python/Chapter-12/Sum.java
public class Sum {
    public static void main(String[] args) {
        System.out.println(sum(6 * 6));
    }

    static int sum(int n) {
	int s = 0;
	for (int i = 0; i <= n; ++i) {
	    s += i;
	}
	return s;
    }
}
